modules/Microsoft.DigitalTwins/digitalTwinsInstances/readme.md

```diff
@@ -0,0 +1,612 @@
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"userAssignedIdentities": {
+    "value": {
+        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001": {},
+        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-002": {}
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+userAssignedIdentities: {
+    '/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001': {}
+    '/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-002': {}
+}
+```
+
+</details>
+<p>
+
+### Parameter Usage: `eventhubEndpoint`
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+        "eventhubEndpoint": {
+            "value": {
+              "authenticationType": "IdentityBased", // IdentityBased or KeyBased
+              "name": "<Endpoint Name>",
+              "entityPath": "evh1", // Event Hub Name
+              "endpointUri": "sb://xyz.servicebus.windows.net", //Event Hub namespace, including sb://
+              "deadLetterUri": "",
+              "deadLetterSecret": "",
+              "connectionStringPrimaryKey": "", //Keybased Auth
+              "connectionStringSecondaryKey": "" //Keybased Auth
+            }
+        }
+```
+
+</details>
+<p>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+  eventhubEndpoint: {
+    authenticationType: 'IdentityBased' // IdentityBased or KeyBased
+    name: '<Endpoint Name>'
+    entityPath: 'evh1' // Event Hub Name
+    endpointUri: 'sb://xyz.servicebus.windows.net' //Event Hub namespace, including sb://
+    deadLetterUri: ''
+    deadLetterSecret: ''
+    connectionStringPrimaryKey: '' //Keybased Auth
+    connectionStringSecondaryKey: '' //Keybased Auth
+  }
+  ```
+
+</details>
+<p>
+
+### Parameter Usage: `eventGridEndpoint`
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+        "eventGridEndpoint": {
+            "value": {
+              "name": "<Endpoint Name>",
+              "accessKey1": "",
+              "accessKey2": "",
+              "TopicEndpoint": "",
+              "deadLetterUri": "",
+              "deadLetterSecret": ""
+            }
+        }
+```
+
+</details>
+<p>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+  eventGridEndpoint: {
+    name: '<Endpoint Name>'
+    accessKey1: ''
+    accessKey2: ''
+    TopicEndpoint: ''
+    deadLetterSecret: ''
+    deadLetterSecret: ''
+  }
+  ```
+
+</details>
+<p>
+
+### Parameter Usage: `serviceBusEndpoint`
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+        "serviceBusEndpoint": {
```

Comment:
Please update the identation of this and the other parameter usage examples

Reply:
Hi @AlexanderSehr

Good spot - Fixed in the latest commit.